TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.BasicBlockForNodeGroup.cs

```diff
@@ -29,10 +29,10 @@ public int GroupIndex
             /// <summary>
             /// Terminal blocks within this Group.
             /// </summary>
-            internal List<BasicBlockForNode> TerminalBlocks
+            public IList<BasicBlock<Node, D>> TerminalBlocks
```

Comment:
Is it required by the analyzer ?

Reply:
The goal here is to have all information from multi branch context available as read only value for any analyzer, because the information is here so we can make it available.